### PR TITLE
CCR-365 support safe filenames for windows environments

### PR DESCRIPTION
**Pull request by jrschumacher**
### Proposed Changes

*Replace this text with an overview of what changed and why.*

### Checklist

*Place an `x` inside the brackets to check off items.*

- [ ] I have [updated the change log](https://github.com/virtru/dev-guide/blob/master/process/merge-and-tag.md#change-log)
- [ ] I have updated the `VERSION` file if merging to `master`
- [ ] I have added tests to cover any new functionality or bugfixes

### CODE REVIEW

File: auditexport/utils.py [hunk 1]
```diff
@@ -106,10 +106,14 @@ def pre_export(fileFormat, records):
     return preparedRecords
 
 
-def exportToJson(pathToFolder, records):
+def exportToJson(pathToFolder, records, safeFilename):
     preparedRecords = pre_export('json', records)
+    date = str(datetime.datetime.utcnow().isoformat())
+    
+    if safeFilename:
+        date = re.sub(r'[^a-zA-Z0-9]+', '-', date)
```
Comment (alkalescent) [inline auditexport/utils.py:114]: Can we make this default for Windows so if condition would be either of the following:
- `if safeFilename or os.name == 'nt'`
- `if safeFilename or platform.system() == 'Windows'` (would have to import platform for this one)

And can we be more obvi that the hyphenated numbers are time by ditching hyphens in favor of letters (h, m, s)?
Like [we do here](https://github.com/virtru/console/blob/32c3c26d8573b96ad692b1243bb3f415fdf32566/src/scenes/AdvancedQuery/AdvancedQuery.js#L30-L32)?

Comment (jrschumacher) [reply auditexport/utils.py:114]: @suchak1 I took your suggestion on the `platform.system()`, but going to defer on the adding letters.

The python script will likely be phased out in favor of API connection or stream connection to the next Audit API. If we end up needing a script of some kind I'd rather rewrite it in Go and build binaries for Windows, *nix, and Macos.

File: auditexport/utils.py [hunk 2]
```diff
@@ -106,10 +106,14 @@ def pre_export(fileFormat, records):
     return preparedRecords
 
 
-def exportToJson(pathToFolder, records):
+def exportToJson(pathToFolder, records, safeFilename):
     preparedRecords = pre_export('json', records)
+    date = str(datetime.datetime.utcnow().isoformat())
+    
+    if safeFilename or platform.system():
```
Comment (alkalescent) [inline auditexport/utils.py]:
```suggestion
    if safeFilename or platform.system() == 'Windows':
```

**Review by alkalescent — APPROVED** [review]
lgtm pending tests
do you know why they fail?

**Pull request merged**